packages/picker/README.md

```diff
@@ -459,6 +459,28 @@ When in pending state, `<sp-picker>` elements will not respond to click events a
 </sp-picker>
 ```
 
+## Force Popover on Mobile Devices
+
+When on a mobile device, `<sp-picker>` elements will render an `<sp-tray>` to the bottom of the viewport by default. To override this functionality so that the picker remains an `<sp-popover>` in mobile, you can simply add the attribute `forcePopover` to the `<sp-picker>`. To see this functionality in action, load this page from your mobile device or use Chrome DevTools (or equivalent) and select a mobile device once the Device Toolbar (the phone/tablet icon) is active.
```

Comment:
nit: This is a very nice technical explanation, but could we also provide guidance focusing on the UX aspects of it?
e.g., taking some inspiration from https://spectrum.adobe.com/page/tray/#Trays-or-popovers? and https://spectrum.adobe.com/page/popover/#Popovers-or-trays?